<?xml version="1.0" encoding="UTF-8"?>
<testng-results retried="3" ignored="3" total="9" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-03T07:50:27 IST" name="Default suite" finished-at="2024-07-03T07:53:13 IST" duration-ms="166471">
    <groups>
      <group name="Smoke">
        <method signature="homePageTest.verifyheaders()[pri:1, instance:practiceAPK.homePageTest@45752059]" name="verifyheaders" class="practiceAPK.homePageTest"/>
        <method signature="homePageTest.validateFormFill()[pri:2, instance:practiceAPK.homePageTest@45752059]" name="validateFormFill" class="practiceAPK.homePageTest"/>
        <method signature="homePageTest.validateToast()[pri:3, instance:practiceAPK.homePageTest@45752059]" name="validateToast" class="practiceAPK.homePageTest"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2024-07-03T07:50:27 IST" name="Default test" finished-at="2024-07-03T07:53:13 IST" duration-ms="166471">
      <class name="practiceAPK.homePageTest">
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:50:27 IST" name="appiumConfig" finished-at="2024-07-03T07:50:47 IST" duration-ms="20431" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method signature="verifyheaders()[pri:1, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:50:47 IST" name="verifyheaders" finished-at="2024-07-03T07:50:52 IST" duration-ms="5035" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyheaders -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:50:52 IST" name="tearDown" finished-at="2024-07-03T07:50:53 IST" duration-ms="328" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:50:53 IST" name="appiumConfig" finished-at="2024-07-03T07:51:13 IST" duration-ms="19709" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method signature="validateFormFill()[pri:2, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:13 IST" name="validateFormFill" finished-at="2024-07-03T07:51:29 IST" duration-ms="16048" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateFormFill -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:29 IST" name="tearDown" finished-at="2024-07-03T07:51:29 IST" duration-ms="398" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:29 IST" name="appiumConfig" finished-at="2024-07-03T07:51:49 IST" duration-ms="20284" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method retried="true" signature="validateToast()[pri:3, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:49 IST" name="validateToast" finished-at="2024-07-03T07:51:54 IST" duration-ms="5221" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter your nam] but found [Please enter your name]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter your nam] but found [Please enter your name]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at learnAppiumFramework.vivek.android.homePage.validateToastMessage(homePage.java:86)
at practiceAPK.homePageTest.validateToast(homePageTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateToast -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:55 IST" name="tearDown" finished-at="2024-07-03T07:51:55 IST" duration-ms="445" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:51:55 IST" name="appiumConfig" finished-at="2024-07-03T07:52:15 IST" duration-ms="19988" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method retried="true" signature="validateToast()[pri:3, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:15 IST" name="validateToast" finished-at="2024-07-03T07:52:20 IST" duration-ms="5232" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter your nam] but found [Please enter your name]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter your nam] but found [Please enter your name]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at learnAppiumFramework.vivek.android.homePage.validateToastMessage(homePage.java:86)
at practiceAPK.homePageTest.validateToast(homePageTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateToast -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:20 IST" name="tearDown" finished-at="2024-07-03T07:52:21 IST" duration-ms="363" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:21 IST" name="appiumConfig" finished-at="2024-07-03T07:52:41 IST" duration-ms="20959" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method retried="true" signature="validateToast()[pri:3, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:41 IST" name="validateToast" finished-at="2024-07-03T07:52:47 IST" duration-ms="5146" status="SKIP">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter your nam] but found [Please enter your name]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter your nam] but found [Please enter your name]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at learnAppiumFramework.vivek.android.homePage.validateToastMessage(homePage.java:86)
at practiceAPK.homePageTest.validateToast(homePageTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateToast -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:47 IST" name="tearDown" finished-at="2024-07-03T07:52:47 IST" duration-ms="324" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="appiumConfig()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:52:47 IST" name="appiumConfig" finished-at="2024-07-03T07:53:08 IST" duration-ms="20949" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- appiumConfig -->
        <test-method signature="validateToast()[pri:3, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:53:08 IST" name="validateToast" finished-at="2024-07-03T07:53:13 IST" duration-ms="5199" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Please enter your nam] but found [Please enter your name]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Please enter your nam] but found [Please enter your name]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at learnAppiumFramework.vivek.android.homePage.validateToastMessage(homePage.java:86)
at practiceAPK.homePageTest.validateToast(homePageTest.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateToast -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:practiceAPK.homePageTest@45752059]" started-at="2024-07-03T07:53:13 IST" name="tearDown" finished-at="2024-07-03T07:53:13 IST" duration-ms="344" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- practiceAPK.homePageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
